### PR TITLE
zig fmt fixes

### PR DESCRIPTION
**Pull request by LakeByTheWoods**
This closes a bunch of fmt issues  
I think the changes are at least justifiable, but am happy to revert things upon getting other's opinions.

### CODE REVIEW

**Comment by LemonBoy** [comment]
If you want to close one more ticket related to `zig fmt` there's #6171 too

**Comment by andrewrk** [comment]
Nice work! I'm going to back out https://github.com/ziglang/zig/pull/6293/commits/ed24adb0e7b8110e41e2db4a1b13b1eae235269e before merging just so that it doesn't cause unnecessary conflicts with several of the open stage2 PRs: #6291 #6255 #6250 #6295
We can run `zig fmt` on these directories at a strategic moment when there is not a bunch of open PRs in those directories.

File: lib/std/build.zig
Comment (andrewrk) [inline lib/std/build.zig:913]: Hmm I definitely prefer the previous formatting here.

File: test/stage2/zir.zig
Comment (andrewrk) [inline test/stage2/zir.zig:159]: I prefer the previous formatting here.

**Review by andrewrk — CHANGES_REQUESTED** [review]

File: lib/std/build.zig
Comment (LakeByTheWoods) [reply lib/std/build.zig:913]: Fair enough. Just to twist your arm a *little* bit, there is an inconstancy between nested `if`s and other nested items (ie. `if`s will follow the style above, but `switch`s will follow the status quo style).
I'll revert the fix, and #6114 should be rejected.

File: test/stage2/zir.zig
Comment (LakeByTheWoods) [reply test/stage2/zir.zig:159]: Fair enough, will update

**Comment by LakeByTheWoods** [comment]
>  I'm going to back out ed24adb

By all means do, that was mostly to show changes in context. Would you like me to push a revert commit, or drop it in a rebase, or is it easier for you to handle it when pulling?

File: lib/std/build.zig
Comment (andrewrk) [reply lib/std/build.zig]: Hmm I see your point (and I'm glad you're pushing back btw). Can we stick with status quo for the purposes of this PR and then discuss it further in that issue?

**Comment by andrewrk** [comment]
> By all means do, that was mostly to show changes in context. Would you like me to push a revert commit, or drop it in a rebase, or is it easier for you to handle it when pulling?

If you don't mind dropping it in a rebase, that will be easiest for me so that I can give this one more lookover and then SLAP that big green button.

File: lib/std/build.zig
Comment (LakeByTheWoods) [reply lib/std/build.zig]: We certainly can

**Comment by LakeByTheWoods** [comment]
Hmm, I'm not sure why the Mac build failed...

**Comment by andrewrk** [comment]
Looks like an operational issue with Azure, not anything on your end. Thanks for the ping - I'll take one final look and then hit that phat merge button

**Pull request merged**
